components/molecule/photoUploader/src/index.js

```diff
@@ -108,14 +108,28 @@ const MoleculePhotoUploader = ({
   })
 
   const _callbackPhotosUploaded = list => {
-    if (list.length) {
-      const blobsArray = list.reduce((array, currentFile) => {
-        const {blob, url, isNew, isModified, hasErrors, file} = currentFile
-        array.push({blob, url, isNew, isModified, hasErrors, file})
-        return [...array]
-      }, [])
-      callbackPhotosUploaded(blobsArray)
-    }
+    const blobsArray = list.reduce((array, currentFile) => {
+      const {
+        blob,
+        url,
+        isNew,
+        isModified,
+        hasErrors,
+        file,
+        preview
+      } = currentFile
+      array.push({
+        blob,
+        url,
+        isNew,
+        isModified,
+        hasErrors,
+        file,
+        previewUrl: preview
```

Comment:
I think is better to expose the original `preview` argument attribute. It can be so much confusing this inner silent naming changes

Reply:
Yes, it is confusing. Is it a good idea to rename `preview` argument everywhere to `previewUrl` ? I believe it's a clearer name than only `preview`. What do you think?